cypress/e2e/auth/auth.cy.js

```diff
@@ -17,12 +17,23 @@ describe('Login Feature', function () {
   it('navigate user to admin panel when logged in', () => {
     const { username, password } = this.authData;
     cy.login(username.valid, password.valid);
-    cy.url().should('contains', '/admin');
   });
 
   it('display error message when entered incorrect credential', () => {
     const { username, password } = this.authData;
     cy.login(username.invalid, password.invalid);
     cy.get('[data-test-id="login.error"]').should('be.visible');
   });
+
+  it('Signup Feature', () => {
+    const { username, email, password } = this.authData;
+    cy.signup(username.valid, email.valid, password.valid);
+    cy.get('[data-testid="checkbox"]').click();
+    cy.get('[data-testid="register"]').click();
+    cy.get('#displayName').type('KevinLu');
+    cy.get('[data-testid="continue-btn"]').click();
+    cy.get(':nth-child(2) > .css-13qzv8t').click();
```

Comment:
need to test this with `data-test-id`

Reply:
The category(cy.get(':nth-child(2) > .css-13qzv8t').click(), ) seems like not a button, how can I click it? I changed the display-name already.